api/routes/ticket.js

```diff
@@ -3,10 +3,17 @@ var router = express.Router();
 var mongoose = require('mongoose');
 var Ticket = require('../models/Ticket');
 
-// Save Ticket 
+// Update Ticket 
+//TODO: doesn't update need id??
```

Comment:
I think you have the `put` and `post` reversed. Usually, you put in a new object and then post to modify it using the ID.